TrackingAntennaV3/main/Magnetometer.hpp

```diff
@@ -1,20 +1,23 @@
-/**
- * @file   Magnetometer.hpp
  * @Author Michael Lenover (WARG)
  * @date   January 5, 2017
  * @brief  Magnetometer initialization functions
+ *     Connection based off of the Arduino ethernet library.
+ *     Uses DHCP so no need to know the IP of the shield once connected.
  *         Connection based off of the Arduino ethernet library.
  *         Uses DHCP so no need to know the IP of the shield once connected.
  * @see https://www.arduino.cc/en/Reference/Ethernet
  */
 
-#ifndef MAGNETOMETER
-#define MAGNETOMETER
-
 #define COMPASS_ID 12345
 
+#define CALIBRATE_MAGNETOMETER false
+
+//Magnetometer data struct
+typedef struct {
+  float x;
+  float y;
+  float z;
+} Magnetometer;
 #define CALIBRATE_MAGNETOMETER true
```

Comment:
I know you're not finished with this, but I feel like we should instead have a `calibrateMagnetometer` function, instead of  a preprocessor statement. If need be this statement can go into the main function, which will execute the calibrateMagnetometer function if need be